data/ecosystems/c/chia.toml

```diff
@@ -2052,3 +2052,291 @@ url = "https://github.com/zodi1337/Raspberry-HDD-Mining-Projects"
 
 [[repo]]
 url = "https://github.com/zolmine/roboCopy"
+
+[[repo]]
+ url ="https://github.com/somediego/kissmp"
+
+[[repo]]
+ url ="https://github.com/L4bb3rs/Chia-CATS-Sent-to-Bags"
+
+[[repo]]
+ url ="https://github.com/hashgreen/docs.hashgreen.github.io"
+
+[[repo]]
+ url ="https://github.com/SlowestTimelord/slowesttimelord.github.io"
+
+[[repo]]
+ url ="https://github.com/KryptomineCH/Dexie.Space.Net"
+
+[[repo]]
+ url ="https://github.com/KryptomineCH/Chia-NFT-Minter"
+
+[[repo]]
+ url ="https://github.com/AppleOfEnlightenment/api"
+
+[[repo]]
+ url ="https://github.com/CryptoLFDM/ChiaRPCWrapper"
+
+[[repo]]
+ url ="https://github.com/Chia-Network/cadt-ui"
+
+[[repo]]
+ url ="https://github.com/greimela/nostr-dex"
+
+[[repo]]
+ url ="https://github.com/gowke/lotus-blockchain"
+
+[[repo]]
+ url ="https://github.com/KryptomineCH/CHIA-RPC"
+
+[[repo]]
+ url ="https://github.com/L4bb3rs/Chia-Py-RPC"
+
+[[repo]]
+ url ="https://github.com/Open-Market-Dev-Club/spriggan-marketplace-admin-dapp"
+
+[[repo]]
+ url ="https://github.com/FireAcademy/telemetry"
+
+[[repo]]
+ url ="https://github.com/bramcohen/chia-gaming"
+
+[[repo]]
+ url ="https://github.com/FireAcademy/prices"
+
+[[repo]]
+ url ="https://github.com/lourkeur/.chia.nix.old"
+
+[[repo]]
+ url ="https://github.com/lmacken/plow"
```

Comment:
Please remove

Reply:
Was this a duplicate?  It is a chia plot mover.

Reply:
my bad, let's keep it